lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -30,14 +50,15 @@ def authenticate_otp(code, options = {})
           totp.verify_with_drift(code, drift)
         end
 
-        def otp_code(time = Time.now, options = {})
+        def generate_totp_code(time = Time.now, options = {})
+          raise "generate_totp_code called with no otp_secret_key set" if otp_secret_key.nil?
```

Comment:
The one reason we would want to leave this method here is to support apps that were sending the TOTP code via SMS or some other method, but I would rather not support bad behavior.